doc/user/content/release-notes.md

```diff
@@ -69,10 +69,11 @@ Wrap your release notes at the 80 character mark.
 - Prevent a bug that would allow `CREATE OR REPLACE` statements to create dependencies
```

Comment:
Is there a reason this entry is "Prevent" while the one above is "Fix"?

Reply:
from the inline style guide on this doc:

> Strive for some variety of verbs. "Support new feature" gets boring as a release note.

Reply:
OK, I will take my complaint to the release notes guidelines, with which I vehemently disagree, and mark this instance as resolved.